common/model/image.ts

```diff
@@ -0,0 +1,23 @@
+import type { Tasl } from './tasl';
```

Comment:
Not sure we need to use import type in `TS`, it's just import.

Reply:
ah ok – good to know. So you can't have a component and type named the same exported from the same file?